### ISSUE TITLE
Issue with terraform state for specific json elements

### ISSUE THREAD

**Issue by tiej-dr** [akamai/terraform-provider-akamai#234]
I'm using "akamai_property" resource, and specifying the property rules in json files, as follows:

```
resource "akamai_property" "api-gateway" {
    name        = var.name
    group_id    = var.group_id
    contract_id = var.contract_id
    product_id  = var.product_id
    hostnames {
      cname_from = "${var.env}.foo.xyz"
      cname_to = "${var.env}.foo.xyz.edgekey.net"
      cert_provisioning_type = var.cert_provisioning_type
    }
    rule_format = "v2020-11-02"
    rules = data.akamai_property_rules_template.rules.json
    lifecycle {
      ignore_changes = [
        production_version,
        staging_version,
        latest_version
      ]
    }
}

data "akamai_property_rules_template" "rules" {
  template_file = abspath("${path.root}/property-snippets/main.json")
  variables {
      name = "name"
      value = "${var.name}"
      type = "string"
  }
  ...
}
```

The json is downloaded via control panel (or using the akamai api), and some elements has an empty list as value, e.g: 
```
"children": []
"criteria": []
"behaviors": []
```

It's seems like these elements is translated into `options = {}` in Terraform state, and results in false positive diffs next time terraform plan is run.

### Terraform Version
Testet with following versions 
0.14.2
0.15.0
0.15.1
 
TF provider: registry.terraform.io/akamai/akamai v1.5.x

### Affected Resource(s)
- akamai_property_rules
- akamai_property_rules_template

### Expected Behavior
When running `terraform plan`, only diffs for the changed values should be shown.

### Actual Behavior
There is a diff on all children/criteria/behaviors elements where value is an empty list (even though this is not changed)

![Issue_with_terraform_state_for_specific_•_Untitled-2_—_devops-api-properties](https://user-images.githubusercontent.com/58421683/116381267-0533ea00-a815-11eb-8d4c-77e3f402b1ca.png)

### Steps to Reproduce
1. Change some value in rules.json 
2. Run `terraform plan`

Note: Running `terraform plan` without any json changes works as expected: "No changes. Infrastructure is up-to-date."

**Comment by lkowalsk-akamai-com** [comment]
Hi @tiej-dr 
This issue should be fixed in v1.6.1

**Comment by tiej-dr** [comment]
Can confirm this i solved in 1.6.1. Cheers!
